decidim-core/config/locales/en.yml

```diff
@@ -1706,3 +1706,6 @@ en:
       previous_title: Link to the previous page
       title: 'Page number: '
       truncate: "&hellip;"
+  from:
```

Comment:
I think this should be in the same `colors_title` block. You can see where's that with `grep -rn colors_title decidim-*/config/locales/en.yml`